Mathlib/Algebra/GroupPower/Basic.lean

```diff
@@ -135,13 +135,13 @@ theorem Commute.mul_pow {a b : M} (h : Commute a b) (n : ℕ) : (a * b) ^ n = a
   · rw [pow_zero, pow_zero, pow_zero, one_mul]
   · simp only [pow_succ, ih, ← mul_assoc, (h.pow_left n).right_comm]
 #align commute.mul_pow Commute.mul_pow
-#align add_commute.add_smul AddCommute.add_smul
+#align add_commute.add_nsmul AddCommute.add_smul
```

Comment:
It turns out these are wrong on the mathlib4 side. Should we fix the names here as well?

Reply:
There is a separate PR #1502 for that (which will unfortunately have a lot of merge conflicts with this one...)